tfconfig/modules/multi-nic-network/main.tf

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+module "network1" {
```

Comment:
Just wondering: is there a way to do `foreach` in here and only have 1 template definition?
Two benefits for that:
1. Less code duplication
2. We'll be able to support customer number of networks (not hardcoded 5 as of now)

Reply:
+1. Will fix it.